test/cypress/integration/widgets/deployments_spec.ts

```diff
@@ -39,9 +39,16 @@ describe('Deployments widget', () => {
             .mockLogin();
     });
 
-    it('should be present in Deployments page', () => {
-        cy.searchInDeploymentsWidget(deploymentId);
-        cy.get('.deploymentSegment h3 [aria-label="Deployment name"]').should('have.text', deploymentName);
+    describe('should be present in Deployments page', () => {
+        it('should be able search by depyloyment ID', () => {
+            cy.searchInDeploymentsWidget(deploymentId);
+            cy.get('.deploymentSegment h3 [aria-label="Deployment name"]').should('have.text', deploymentName);
+        });
+
+        it('should be able by deployment name', () => {
+            cy.searchInDeploymentsWidget(deploymentName);
+            cy.get('.deploymentSegment h3 [aria-label="Deployment name"]').should('have.text', deploymentName);
+        });
```

Comment:
I suggest to adjust the naming a bit.
```suggestion
    describe('should be present in Deployments page and', () => {
        it('should allow to search by depyloyment ID', () => {
            cy.searchInDeploymentsWidget(deploymentId);
            cy.get('.deploymentSegment h3 [aria-label="Deployment name"]').should('have.text', deploymentName);
        });

        it('should allow to search by deployment name', () => {
            cy.searchInDeploymentsWidget(deploymentName);
            cy.get('.deploymentSegment h3 [aria-label="Deployment name"]').should('have.text', deploymentName);
        });
```

WDYT?

Optional:

Seeing the same 2 lines, maybe also common function for 
```
cy.get('.deploymentSegment h3 [aria-label="Deployment name"]').should('have.text', deploymentName);
```
can be extracted. It's optional IMO.

Reply:
applied: https://github.com/cloudify-cosmo/cloudify-stage/pull/2079/commits/40a4652300c209b1746a3947495475d338092e48